azure/README.md

```diff
@@ -6,7 +6,6 @@
 
 ```bash
 export RESOURCE_GROUP="REPLACE_ME"
-export LOCATION="REPLACE_ME"
```

Comment:
The subsequent command will fail to create resource group without providing location?